examples/lb3-application/README.md

```diff
@@ -337,13 +337,203 @@ $ npm start
+
+  ```ts
+  // can use lb4's testlab's supertest as the dependency is already installed
+  const {supertest} = require('@loopback/testlab');
+  const assert = require('assert');
+  const should = require('should');
+  const {CoffeeShopApplication} = require('../../dist/application');
+
+  let app;
+
+  function jsonForLB4(verb, url) {
+    // use the lb4 app's rest server
+    return supertest(app.restServer.url)
+      [verb](url)
+      .set('Content-Type', 'application/json')
+      .set('Accept', 'application/json')
+      .expect('Content-Type', /json/);
+  }
+  ```
+
+- 2.2 Boot and start the LoopBack 4 app in your before hook, and stop the app in
+  the after hook:
+
+  ```ts
+  describe('LoopBack 3 style tests - launch LB4 app', function () {
+    before(async function () {
+      app = new CoffeeShopApplication();
+      await app.boot();
+      await app.start();
+    });
+
+    after(async () => {
+      await app.stop();
+    });
+
+    // your tests here
+    runTests();
+  });
+  ```
+
+Example of this use can be seen in
+[`test/acceptance.js`](https://github.com/strongloop/loopback-next/tree/master/examples/lb3-application/lb3app/test/acceptance.js)
+which has the same tests as
+[`src/__tests__/acceptance/lb3app.acceptance.ts`](https://github.com/strongloop/loopback-next/blob/spike/lb3test/examples/lb3-application/src/__tests__/acceptance/lb3app.acceptance.ts),
+but in LB3 style. And
+[`test/authentication.js`](https://github.com/strongloop/loopback-next/tree/master/examples/lb3-application/lb3app/test/authentication.js)
+
+Now when you run `npm test` your LoopBack 3 tests should be run along with any
+LoopBack 4 tests you have.
+
+Optional: Another option is to migrate your tests to use LoopBack 4 style of
+testing, similar to `src/__tests__/acceptance/lb3app.acceptance.ts`.
+Documentation for LoopBack testing can be found in
+https://loopback.io/doc/en/lb4/Testing-your-application.html.
+
+## Running Integration Tests
+
+For the integration tests, LoopBack 3 models were bound to the LoopBack 4
+application in order to allow JavaScript API to call application logic such as
+`Model.create()`. This can be seen in
+[`packages/booter-lb3app/src/lb3app.booter.ts`](https://github.com/strongloop/loopback-next/blob/spike/lb3test/packages/booter-lb3app/src/lb3app.booter.ts#L76-L85).
+
+In order to retrieve the model from the application's context, `get()` can be
+used as follows:
+
+```ts
+describe('LoopBack 3 style integration tests', function () {
+  let app;
+  let CoffeeShop;
+
+  before(async function () {
+    // If launch the LoopBack 4 application
+    // app = new CoffeeShopApplication();
+    app = new ExpressServer();
+    await app.boot();
+    await app.start();
+  });
+
+  before(() => {
+    // follow the syntax: lb3-models.{ModelName}
+    // If launch the LoopBack 4 application
+    // CoffeeShop = await app.get('lb3-models.CoffeeShop');
+    CoffeeShop = await app.lbApp.get('lb3-models.CoffeeShop');
```

Comment:
How did you get the syntax `lb3-models.{ModelName}`?  Is `lb3-modesl` the name of this LB3 model collection?

Reply:
It's implemented in this PR https://github.com/strongloop/loopback-next/pull/5361, in story https://github.com/strongloop/loopback-next/issues/5299

Let me update the naming convention in doc.